source/docs/casper/concepts/session-code.md

```diff
@@ -1,3 +1,8 @@
+---
+title: Session Code
+slug: /concepts/session-code
+---
+
 # Contracts and Session Code
```

Comment:
That makes sidebar title to be _Session Code_, but page header is still _**Contracts** and Session Code_. I would simply update title to avoid _Contract_ word:

```suggestion
# Session Code
```

Reply:
The page contains informaiton about both things, right? the file on disk was named just session-code, i believe that is why the sidebar is just session code, it's legacy.. I would go the other way, properly rename the title to "Contracts and Session Code"

Reply:
From my perspective this page is about **session code** - which is almost like contract, that is why differences are explained.